cache-tests.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var Cache = require('../cache');
```

Comment:
> is my git showing something different from yours 

Btw I get the same problems when I do `hub checkout https://github.com/broccolijs/broccoli-filter/pull/26` (instead of `merge`), so it's not a merge hiccup. Maybe did you point your branch to a different commit than you meant to?